src/renderer/components/ProductTour/Dashboard/Card.js

```diff
@@ -0,0 +1,132 @@
+// @flow
+
+import React, { useCallback, useContext } from "react";
+import WrappedCard from "~/renderer/components/Box/Card";
+import { Trans } from "react-i18next";
+import Text from "~/renderer/components/Text";
+import Box from "~/renderer/components/Box";
+import Image from "~/renderer/components/Image";
+import styled from "styled-components";
+import IconLock from "~/renderer/icons/Lock";
+import IconCheckFull from "~/renderer/icons/CheckFull";
+import ProductTourContext from "~/renderer/components/ProductTour/ProductTourContext";
+import { useDispatch } from "react-redux";
+import useTheme from "~/renderer/hooks/useTheme";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+
+const StyledCard: ThemedComponent<{}> = styled(WrappedCard)`
+  height: 172px;
+  padding: 16px;
+  color: "white";
+  background-color: ${p =>
+    p.completed ? p.theme.colors.identity : p.theme.colors.palette.primary.main};
+  opacity: ${p => (p.disabled ? 0.4 : 1)};
+  cursor: ${p => (p.disabled || p.completed ? "default" : "pointer")};
+  position: relative;
+  overflow: hidden;
+  & ${Box}, & ${Text} {
+    z-index: 2;
+  }
+`;
+
+const StyledImage: ThemedComponent<{}> = styled(Image)`
+  z-index: 1;
+  max-width: 90px;
+  position: absolute;
+  right: 10px;
+  bottom: -10px;
+  mix-blend-mode: luminosity;
+`;
+
+const Circle: ThemedComponent<{}> = styled.div`
+  height: 16px;
+  width: 16px;
+  border-radius: 16px;
+  border: 1px solid white;
+`;
+
+const Card = ({
+  require,
+  appFlow,
+  onBeforeFlow,
+  onAfterFlow,
+  title,
+  titleCompleted,
+  overrideContent,
+  illustration,
+  controlledModals,
+  learnMoreCallback,
+  ...params
+}: {
+  require?: string,
+  appFlow: string,
+  onBeforeFlow: () => void,
+  onAfterFlow: () => void,
+  title: React$Node,
+  titleCompleted: React$Node,
+  overrideContent?: boolean,
+  illustration: *,
+  controlledModals?: string[],
+  learnMoreCallback?: () => void,
+}) => {
+  const { state, send } = useContext(ProductTourContext);
+  const theme = useTheme();
+  const dispatch = useDispatch();
+
+  const {
+    context: { completedFlows },
+  } = state;
+  const disabled = require && !completedFlows.includes(require);
+  const completed = completedFlows.includes(appFlow);
+
+  const onClick = useCallback(() => {
+    // if (!disabled && !completed) {
```

Comment:
what are those comments ? 

Same for the one underneath (else if)

Reply:
Because it was a wip and I used it to bypass the access control of the cards